### PR TITLE
update DEVELOPING and IMPLEMENTATION docs with new ossmc info

### PR DESCRIPTION
**Pull request by jmazzitelli**

### CODE REVIEW

**Review by ferhoyos — APPROVED** [review]
LGTM

File: IMPLEMENTATION.adoc [hunk 1]
```diff
@@ -2,65 +2,70 @@
...
 
-The KO can support installing multiple versions of the Kiali server by invoking different versions of the Ansible roles. The versions the KO supports are defined in link:https://github.com/kiali/kiali-operator/blob/v1.69.0/playbooks/default-supported-images.yml[default-supported-images.yml]. For each version supported, there is an link:https://github.com/kiali/kiali-operator/tree/v1.69.0/roles[Ansible role] that the KO executes when it needs to install or remove that Kiali version.
+The KO can support installing multiple versions of the Kiali server and OSSMC by invoking different versions of the Ansible roles. The versions the KO supports are defined in link:https://github.com/kiali/kiali-operator/blob/master/playbooks/kiali-default-supported-images.yml[kiali-default-supported-images.yml] and (for OSSMC) in link:https://github.com/kiali/kiali-operator/blob/master/playbooks/ossmconsole-default-supported-images.yml[ossmconsole-default-supported-images.yml]. For each version supported, there is an link:https://github.com/kiali/kiali-operator/tree/master/roles[Ansible role] that the KO executes when it needs to install or remove that Kiali or OSSMC version.
```
Comment (jshaughn) [inline IMPLEMENTATION.adoc:17]: The Kiali and OSSMC versions must be the same, right, they are linked?

File: IMPLEMENTATION.adoc [hunk 2]
```diff
@@ -2,65 +2,70 @@
 
 This document describes the current state of the implementation of the Kiali Operator (henceforce, "KO"). The purpose for this document is to provide an introduction to the internals of the KO to those who have a need to modify or enhance the KO.
 
-NOTE: This document reflects the current implementation as of release link:https://github.com/kiali/kiali-operator/tree/v1.69.0[v1.69.0]
+NOTE: This document reflects the current implementation as of release link:https://github.com/kiali/kiali-operator/tree/v1.76.0[v1.76.0]
```
Comment (jshaughn) [inline IMPLEMENTATION.adoc:5]: This branch doesn't exist yet.  In general, some entries here are for 1.76, some for 1.75 and some for master, is that all intentional?

Comment (jmazzitelli) [reply IMPLEMENTATION.adoc:5]: right, I just put that in there for now- we'll get that branch in a week

File: IMPLEMENTATION.adoc (hunk 1 above)
Comment (jmazzitelli) [reply IMPLEMENTATION.adoc:17]: yeah, today they need to be linked.

**Pull request merged**